lib/ansible/module_utils/docker_common.py

```diff
@@ -416,6 +420,58 @@ def _handle_ssl_error(self, error):
                       % (self.auth_params['tls_hostname'], match.group(1), match.group(1)))
         self.fail("SSL Exception: %s" % (error))
 
+    def _get_minimal_versions(self, option_minimal_versions, ignore_params=None):
+        self.option_minimal_versions = dict()
+        for option, data in self.module.argument_spec.items():
```

Comment:
Maybe I should have used
```suggestion
        for option in self.module.argument_spec:
```
:-)